kythe/cxx/indexer/textproto/plugin.h

```diff
@@ -54,6 +54,11 @@ class PluginApi {
   // required_inputs.
   virtual proto::VName VNameForRelPath(
       absl::string_view simplified_path) const = 0;
+
+  // Returns a reference to the descriptor pool built from the .proto files
```

Comment:
A minor nit (since "reference" is a term of art in the type system):

```suggestion
  // Returns a pointer to the descriptor pool built from the .proto files
```

Reply:
:+1: